### PR TITLE
News schema

### PR DESCRIPTION
**Pull request by Karyum**
rel #95

### CODE REVIEW

File: database/news_schema.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
+
+const newsSchema = mongoose.Schema({
+  newsTitle: {type: String, required: true},
```
Comment (esraajb) [inline database/news_schema.js]: no need for news, just title

File: database/news_schema.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  subhead: {type: String, required: true},
```
Comment (esraajb) [inline database/news_schema.js]: maybe subtitle as you are using 'title' on line 4?

File: database/news_schema.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  newsDescription: {type: String, required: true},
```
Comment (esraajb) [inline database/news_schema.js]: just description

File: database/news_schema.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  author: {type: String, required: true},
+  newsDate: {type: Date, default: Date.now}
```
Comment (esraajb) [inline database/news_schema.js]: date

File: database/news_schema.js (hunk 1 above)
Comment (Karyum) [inline database/news_schema.js]: wouldn't it make our code more readable in the hbs files ? cause event schema also has title and date ....??

Comment (Karyum) [inline database/news_schema.js]: @esraajb

Comment (esraajb) [inline database/news_schema.js]: it's the news file, so it should be clear enough.
anyway, if you feel ok with how it is now, it's just fine

Comment (Karyum) [inline database/news_schema.js]: we are going to keep it like that for clarity :wink:

**Pull request merged**